### PR TITLE
feat: hide application search tips behind (?) icon

### LINKED ISSUE #1767: Hide "example searches" from Applications page
Add a question mark that shows the current "Example searches: .... Learn more" line

Also make the search box a bit wider

### PR DESCRIPTION
**Pull request by opqdonut**
closes #1767

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue
- [x] consider adding screenshots for ease of review

## Documentation
- [x] update changelog if necessary

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [x] no critical TODOs left to implement

### CODE REVIEW

**Comment by opqdonut** [comment]
Collapsed:
tips-collapsed
Expanded:
tips-expanded

**Comment by opqdonut** [comment]
Collapsed:
tips-collapsed
Expanded:
tips-expanded

File: src/cljs/rems/search.cljs [hunk 1]
```diff
@@ -70,49 +70,55 @@
...
+        [:a.application-search-tips.btn.btn-link.collapsed
+         {:data-toggle "collapse"
+          :href (str "#" collapse-id)
+          :aria-label (text :t.search/example-searches)
+          :aria-expanded "false"
+          :aria-controls collapse-id}
+         [:i.fa.fa-question-circle]]
+        (when searching?
+          [spinner/small])]
+       [:div.search-tips.collapse {:id collapse-id}
```
Comment (luontola) [inline src/cljs/rems/search.cljs]: This needs to be focused (for screen readers) when the button to show it is clicked.

File: src/cljs/rems/search.cljs [hunk 2]
```diff
@@ -70,49 +70,55 @@
...
-             :on-click (fn []
-                         (reset! input-value "")
-                         (on-search "")
-                         (.focus @input-element))}
-            [close-symbol]]])]
-
-       (when searching?
-         [spinner/small])])))
-
-(defn application-search-tips []
```
Comment (luontola) [inline src/cljs/rems/search.cljs]: The same `search-field` component is also used on the catalogue page and admin pages, where the application search tips don't apply.

Comment (opqdonut) [reply src/cljs/rems/search.cljs]: right! done

File: src/cljs/rems/search.cljs (hunk 1 above)
Comment (opqdonut) [reply src/cljs/rems/search.cljs]: done

**Pull request merged**